pep-0708.rst

```diff
@@ -699,147 +829,31 @@ common for people to use their own registries, but in Python we encourage you to
 do just that.
 
 
-Open questions
+Open Questions
 ==============
 
 1. The `original proposal document <https://docs.google.com/document/d/184fQkb6NggVQfYmjTDA7p_U3iWDKk6grc2DigT1X3Es/>`__
```

Comment:
```suggestion
*  The `original proposal document <https://docs.google.com/document/d/184fQkb6NggVQfYmjTDA7p_U3iWDKk6grc2DigT1X3Es/>`__
```

You might not want to number these, since there's only one item currently, there isn't an unambiguous order and it would be easy for items to get out of sync when adding, removing or changing items.